src/message.ts

```diff
@@ -414,6 +414,59 @@ export class Message implements Sayable {
     return fromId === userId
   }
 
+  /**
+   *
+   * Get the mentioned contact which the message is mentioned for.
+   * @returns {Contact[]} return the contactList which the message is mentioned for
+   *
+   * @example
+   * ```ts
+   * const content = message.content()
+   * const contactList = message.mention()
+   * console.log(content)
+   * console.log(contactList)
+   * ```
+   */
+  public mention(): Contact[] {
```

Comment:
I vote for `mentioned()`
`mention` can be a noun, but most time it remind me `mention(somebody)`
wechaty api are simple, like `user.self()` instead of `user.isSelf()`, so `getMentioned` is not wechaty style.
but I think `metioned` is fine, it's a adj, can be who are metioned is this room. will not confused me.